packages/datagateway-search/src/searchPageTable.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Box from '@material-ui/core/Box';
+import { Paper } from '@material-ui/core';
+import { StateType } from './state/app.types';
+import { connect } from 'react-redux';
+import InvestigationSearchTable from './table/investigationSearchTable.component';
+import DatasetSearchTable from './table/datasetSearchTable.component';
+import DatafileSearchTable from './table/datafileSearchTable.component';
+
+interface SearchTableStoreProps {
+  requestReceived: boolean;
+  datafile: number[];
+  dataset: number[];
+  investigation: number[];
+  datasetTab: boolean;
+  datafileTab: boolean;
+  investigationTab: boolean;
+}
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: string;
+  value: string;
+}
+
+function TabPanel(props: TabPanelProps): React.ReactElement {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <Box
+      component="div"
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      border={0}
+      {...other}
+    >
+      {value === index && <Box p={3}>{children}</Box>}
+    </Box>
+  );
+}
+
+function a11yProps(index: string): React.ReactFragment {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+// eslint-disable-next-line
+const SearchPageTable = (props: SearchTableStoreProps): React.ReactElement => {
+  const [value, setValue] = React.useState('investigation');
+
+  useEffect(() => {
+    let newState = 'investigation';
+    if (!props.investigationTab) {
+      if (props.datasetTab) {
+        newState = 'dataset';
+      } else {
+        if (props.datafileTab) {
+          newState = 'datafile';
+        } else {
+          newState = 'none';
+        }
+      }
+    }
+    setValue(newState);
+  }, [props.investigationTab, props.datasetTab, props.datafileTab]);
+
+  const handleChange = (
+    event: React.ChangeEvent<unknown>,
+    newValue: string
+  ): void => {
+    setValue(newValue);
+  };
+
+  if (props.requestReceived) {
+    return (
+      <div>
+        <AppBar position="static">
+          <Tabs
+            value={value}
+            onChange={handleChange}
+            aria-label="simple tabs example"
```

Comment:
This `aria-label` should be more descriptive of what these tabs are, although I guess this will be fixed in the i18n PR